heidrun/smithsonian.rb

```diff
@@ -0,0 +1,446 @@
+  'photographer and/or colorist',
+  'photographer or collector',
+  'photographer?',
+  'photographerl',
+  'photographerphotographer',
+  'photographers',
+  'photographers?',
+  'photographer}',
+  'photographic firm',
+  'photogrpaher',
+  'playwright',
+  'poet',
+  'possible maker',
+  'printer',
+  'printmaker',
+  'producer',
+  'recordig artist',
+  'recording artist',
+  'recording artist; composer',
+  'recordist',
+  'recordng artist',
+  'sculptor',
+  'shipbuilder',
+  'shipbuilders',
+  'shipping firm',
+  'weaver',
+  'weaver or owner'
+]
+
+def creator?(value)
+  CREATOR_LABEL_VALUES.include?(value)
+end
+
+clean_language = lambda do |i|
+  i.value.gsub(/ languages?/i, '')
+end
+
+is_shown_at_uri = lambda do |i|
+  "http://collections.si.edu/search/results.htm?q=record_ID%3A#{i.value}&repo=DPLA"
+end
+
+extract_format = lambda do |record|
+  format = record['freetext'].field('physicalDescription')
+                             .match_attribute(:label) { |label|
+    ['Physical description', 'Medium'].include?(label)
+  }
+
+  format.concat(record['indexedStructured'].field('object_type'))
+end
+
+# Number of records found ...
+# Accession #         633
+# Catalog #             0
+# Accession Number 490639
+# accession number  66758
+# Catalog Number        0
+# catalog number    67004
+identifier_map = lambda do |record|
+  identifier = record['freetext'].field('identifier')
+                                 .match_attribute(:label) { |label|
+                                   ['Accession #', 'Accession Number',
+                                    'accession number',
+                                    'Catalog #', 'Catalog Number',
+                                    'catalog number'].include?(label)
+                                 }
+
+  identifier.concat(record['descriptiveNonRepeating'].field('record_ID'))
+end
+
+# "Credit line" should be "Credit Line"
+rights_map = lambda do |record|
+  rights = record['descriptiveNonRepeating']
+    .field('online_media', 'media', '@rights')
+
+  if rights.empty?
+    rights = record['freetext'].field('creditLine')
+      .match_attribute(:label, 'Credit Line')
+  end
+
+  rights.concat(record['freetext'].field('objectRights')
+                  .match_attribute(:label, 'Rights'))
+end
+
+# dcterms:subject
+#   <freetext category="topic" label="Topic">;
+#   <freetext category="culture" label="Nationality">;
+#   <topic>;<name>;<culture>;<tax_kingdom>; <tax_phylum>; <tax_division>;
+#   <tax_class>; <tax_order>; <tax_family>;  <tax_sub-family>;
+#   <scientific_name>; <common_name>;<strat_group>; <strat_formation>;
+#   <strat_member>
+#
+#   n at least one record
+#   (http://content9.qa.dp.la/qa/compare?id=825ca339b107da76b17a1ba49f3e92fe ),
+#   there are @label values of "subject" and "event" which seem like they
+#   should also be mapped to Subject.
+#
+# This produces duplicate subjects on some records
+#   - an enrichment will be required to de-dup
```

Comment:
I mean is the enrichment already in an enrichment profile (in the gist) or is there a ticket for it to be added?